interfaces/mount/entry.go

```diff
@@ -184,10 +184,7 @@ func LoadFSTab(reader io.Reader) ([]Entry, error) {
 func SaveFSTab(writer io.Writer, entries []Entry) error {
 	var buf bytes.Buffer
 	for i := range entries {
-		if _, err := buf.WriteString(entries[i].String()); err != nil {
-			return err
-		}
-		if _, err := buf.WriteRune('\n'); err != nil {
+		if _, err := fmt.Fprintf(&buf, "%s\n", entries[i].String()); err != nil {
```

Comment:
The `.String()` in `entries[i]` is not needed, the `%s` will run this implicitly.